apps/web/components/booking/pages/BookingPage.tsx

```diff
@@ -304,6 +309,19 @@ const BookingPage = ({ eventType, booking, profile }: BookingPageProps) => {
                 <h1 className="mb-4 text-3xl font-semibold text-gray-800 dark:text-white">
                   {eventType.title}
                 </h1>
+                {eventType.seatsPerTimeSlot && (
+                  <p
+                    className={`${
+                      booking && booking.attendees.length / eventType.seatsPerTimeSlot >= 0.5
+                        ? "text-rose-600"
+                        : "text-emerald-400"
```

Comment:
NIT: I think this works for small sizes, but what about 100 attendees on 200 seatsPerTimeSlots, would be okay mark  it as red when it's on his 50% capacity, and 100 seats are available? Also I think we should do yellow